apps/shopify/src/basePagination.js

```diff
@@ -70,7 +71,10 @@ export default class BasePagination {
    * and now want to render the next page.
    */
   async _fetchNextPage(products) {
-    return (await this.shopifyClient.fetchNextPage(products)).model;
+
+    const response = (await this.shopifyClient.fetchNextPage(products)).model;
+    const nextProducts = response.map((res) => { return { ...res, id: Buffer.from(res.id).toString('base64') } })
+    return nextProducts
```

Comment:
No idea how shopify ids are built, but you could consider converting the string to utf8, if you're trying to stay safe from a non-strictly binary strings. 

I would anyway use `btoa` here, and not rely on the polyfill for `Buffer` which seems unnecessary

Reply:
Yes I have changed it and will push it in some time as it is still in progress